packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -88,7 +88,8 @@ export interface ProfileHeaderProps {
   sharedQuery?: FlamegraphRendererProps['sharedQuery'];
 }
 
-const Divider = () => <div className={styles.divider} />;
+const Divider = ({ visible = true }: { visible?: boolean }) =>
```

Comment:
how about ?
```suggestion
const Divider = ({ visible = true }) =>
```

Reply:
I think it's better to remove all unnecessary props from Divider at all